### PR TITLE
images/schema + pageInfo

### PR DESCRIPTION
**Pull request by raymondng**
- PageInfo table to hold some metadata about uri
  -- currently holds a strict (simple) subset of information returned (primarily) by embedly API
  -- we _could_ put the extra info in normalized_uri but the nuri table is already heavily used, so I created a new table/repo
  -- this should/will allow us to remove screenshotUpdatedAt from normalized_uri (see separate pull request)

cc: @eishay @andrewconner @ymatsuda @leogrim

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import play.api.libs.functional.syntax._
+import com.keepit.common.db._
+import org.joda.time.DateTime
+import com.keepit.common.time._
+
+trait PageSafetyInfo {
+  def safe:Option[Boolean]
+}
+
+trait PageMediaInfo {
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala:13]: Not used?

File: kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+trait PageGenericInfo {
+  def description:Option[String]
+  // def content:Option[String]
+  def faviconUrl:Option[String]
+}
+
+object PageInfoStates extends States[PageInfo]
+
+case class PageInfo(
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala:25]: Do we store the image URLs anywhere? Or is this an unrelated table?

File: kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala (hunk 1 above)
Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala:13]: Thanks. Please see (updated) EmbedlyExtractResponse -- that's missing from original pull request.

File: kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala (hunk 2 above)
Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/model/PageInfo.scala:25]: Not (yet) -- that's on the list of things to look into.

**Pull request merged**